spring-cloud-dataflow-server/docker-compose-prometheus-rsocket.yml

```diff
@@ -0,0 +1,59 @@
+version: '3'
```

Comment:
Since RSocket is just an implementation detail, can we simply just rename the file to `docker-compose-prometheus.yml`?

Reply:
sure